### ISSUE TITLE
New Features to be added

### ISSUE THREAD

**Issue by mohittk** [space-voyager-21/space-voyager#1]
Waiting for contributers to add new ideas for this website

**Comment by srijit2002** [comment]
we can use [NASA Open Apis](https://api.nasa.gov/) to implement other useful information in the website like we can use picture of the day Api [apod](https://github.com/nasa/apod-api) to include that image in landing page of our webiste

**Comment by mohittk** [comment]
> we can use [NASA Open Apis](https://api.nasa.gov/) to implement other useful information in the website like we can use picture of the day Api [apod](https://github.com/nasa/apod-api) to include that image in landing page of our webiste

**Comment by mohittk** [comment]
> we can use [NASA Open Apis](https://api.nasa.gov/) to implement other useful information in the website like we can use picture of the day Api [apod](https://github.com/nasa/apod-api) to include that image in landing page of our webiste

Yes! that's a great idea. It will be suitable one for landing page.

**Comment by srijit2002** [comment]
@mohittk  can you assign it to me?

**Comment by mohittk** [comment]
> @mohittk can you assign it to me?

Yes! Go for it, but before that! just inform me what changes you are planning to make here only, in organized way.

**Comment by srijit2002** [comment]
Please create a discord server so that I(and other contributors) can talk to you else this comment section will be messy!

**Comment by mohittk** [comment]
> Please create a discord server so that I(and other contributors) can talk to you else this comment section will be messy!

Yeah! For sure. Join with this link. https://discord.gg/YU8H62rS

**Comment by DurgeshKr2242** [comment]
is someone attending on this issue? If not then can you reassign this one to me?

**Comment by mohittk** [comment]
> is someone attending on this issue? If not then can you reassign this one to me?

Do you have any idea for new feature in existing website.

**Comment by DurgeshKr2242** [comment]
I was thinking about adding a different route for a list of all the comets

**Comment by mohittk** [comment]
> I was thinking about adding a different route for a list of all the comets

You mean to say for comets API?

**Comment by DurgeshKr2242** [comment]
Yes
